File: papi/lista2/exercicio5.py
print("Aqui converteremos um valor em metros para centímetros.")

while True:
    try:
        medida_metros = float(input("Por favor favor informe a medida em metros - use o ponto como separador decimal: "))
        break
    except ValueError:
        print("Por favor informe um número válido.")

print("A medida convertida para cebtímetros é", medida_metros * 100)

File: papi/lista2/exercicio5.js
// execute o npm install para instalar uma dependência externa
// estou usando o softio

const softio = require( 'softio' );
let medidaMetros;


async function principal() {
    while (true) {
        medidaMetros = parseFloat(await softio.input( 'Por favor insira o valor em metros: ' ));

        if (medidaMetros > 0) {
            break;
        }

        console.log("Por favor forneça um número maior que zero");
    }

    console.log("A medida informada, convertida para centímetros é:", medidaMetros * 100, "cm");
}

principal();
